apps/routing_transparency.go

```diff
@@ -44,11 +44,11 @@ var _ = AppsDescribe("Routing Transparency", func() {
 	})
 
 	It("appropriately handles certain reserved/unsafe characters", func() {
-		curlResponse := helpers.CurlApp(Config, appName, "/requesturi/!~^'()$\"?!'()$#!'")
```

Comment:
Better solution: Check if `GetIncludeHTTP2Routing()` is `false` and only then run the double-quote test.

Reply:
Done, please review if our intention is correct.

Reply:
That would skip the whole test if HTTP/2 is enabled. What I meant is just skipping the " test and only enable it if HTTP/2 is disabled. Something like this (not tested):
```
	It("appropriately handles certain reserved/unsafe characters", func() {
		curlResponse := helpers.CurlApp(Config, appName, "/requesturi/!~^'()$?!'()$#!'") // no " in request uri
		Expect(curlResponse).To(ContainSubstring("Request"))

		By("preserving all characters")
		Expect(curlResponse).To(ContainSubstring("/requesturi/!~^'()$"))
		Expect(curlResponse).To(ContainSubstring("Query String is [!'()$]"))

		By("preserving double quotes (not for HTTP/2)")
		if !Config.GetIncludeHTTP2Routing() {
			curlResponse = helpers.CurlApp(Config, appName, "/requesturi/\"")
			Expect(curlResponse).To(ContainSubstring("/requesturi/\""))
		}
	})
```

Reply:
The proposal above is tested locally and works as expected.